xbmc/Application.cpp

```diff
@@ -2778,6 +2779,26 @@ bool CApplication::OnMessage(CGUIMessage& message)
     }
     break;
 
+    case GUI_MSG_PLAY_TRAILER:
+    {
+      const CFileItemPtr item = std::static_pointer_cast<CFileItem>(message.GetItem());
+      CFileItem* trailerItem =
+          ContentUtils::GeneratePlayableTrailerItem(item, g_localizeStrings.Get(20410));
```

Comment:
I would write a comment making clear that the item instance is not deleted here by intention.